### PR TITLE
Use correct headline levels

### PR DESCRIPTION
**Pull request by cheeZery**
We shouldn't just use h3 and h5 but h1 and h2 instead, altough it may be a bit larger

### CODE REVIEW

File: src/Writer/MarkdownWriter.php [hunk 1]
```diff
@@ -94,7 +94,7 @@ protected function createPackageLockedLine(
         PackageManagerPackageInterface $package,
         DependencyData $dependency
     ): string {
-        $line = "##### {$package->getName()} `{$package->getVersion()}` {$dependency->getLockSymbol()}";
+        $line = "# {$package->getName()} `{$package->getVersion()}` {$dependency->getLockSymbol()}";
```
Comment (GodLesZ) [inline src/Writer/MarkdownWriter.php]: This seem's to bei wrong, nor?
In `createPackageLine` below you're using **h2**?

File: DEPENDENCIES.md [hunk 2]
```diff
@@ -1,30 +1,30 @@
 ### Composer
```
Comment (GodLesZ) [inline DEPENDENCIES.md]: Isn't this a bit inconsistent?
IMHO the package manager should ne the highest order and his packages are one level smaller.

Now we have an **h3** for manager, **h2** for normal packages and **h1** for locked one?

File: src/Writer/MarkdownWriter.php (hunk 1 above)
Comment (cheeZery) [reply src/Writer/MarkdownWriter.php]: Excuse this fail of a PR. I thought I could fix this real quick in the GitHub online editor without checking out the branch :man_facepalming: :cake: will fix this!

File: DEPENDENCIES.md (hunk 2 above)
Comment (cheeZery) [reply DEPENDENCIES.md]: Thanks for pointing it out, is fixed now! And even the tests shouldn't fail anymore..

**Comment by GodLesZ** [comment]
It's 🍰 time then 🍺

File: src/Parser/MarkdownParser.php [hunk 3]
```diff
@@ -33,7 +33,7 @@ public function getDocumentedDependencies(
 
             $line = ltrim($line);
 
-            if (preg_match("/^#{3}\s(?<packageManagerName>\w+)/", $line, $matches)) {
+            if (preg_match("/^#{1}\s(?<packageManagerName>\w+)/", $line, $matches)) {
```
Comment (GodLesZ) [inline src/Parser/MarkdownParser.php]: This ist not needed in gerneral but also deosn't hurt

**Comment by cheeZery** [comment]
Had to fix it again. My IDE stripped some whitespace at the end of a line in some test data that were actually expected. I fixed the test by removing the whitespace from the assertion. Do you guys thing that was okay?

**Comment by GodLesZ** [comment]
> Had to fix it again. My IDE stripped some whitespace at the end of a line in some test data that were actually expected. I fixed the test by removing the whitespace from the assertion. Do you guys thing that was okay?

AFAIK this test was correct and expected
The rule was to keep all user input,. even added whitespace on the lines (as two whitespace at the end of a line have a special markdown meaning; see [this SO post](https://meta.stackexchange.com/questions/40976/what-is-the-reason-for-the-top-secret-two-space-newline-markdown-weirdness))

So we currently didn't test that part

**Pull request merged**